code/modules/mob/living/carbon/life.dm

```diff
@@ -21,6 +21,7 @@
 
 	//Updates the number of stored chemicals for powers
 	handle_changeling()
+	handle_devil()

```

Comment:
Does it actually need periodic updates ? Souls are only gained/lost in very specific way aren't they ?